.golangci.yml

```diff
@@ -0,0 +1,44 @@
+service:
+  golangci-lint-version: 1.19.1
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - errcheck
+    - gochecknoinits
+    - goconst
+    - gocritic
+    - gocyclo
+    - gofmt
+    - goimports
+    - golint
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - lll
+    - misspell
+    - nakedret
+    - scopelint
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unused
+    - varcheck
+    - whitespace
+    - funlen
+linters-settings:
+  errcheck:
+    check-blank: true
+  funlen:
+    lines: 146
+    statements: 64
```

Comment:
Avoid arbitrary limits. If there's a reason for it, provide a comment.

Reply:
removed